src/lib/asset-saver.js

```diff
@@ -72,6 +72,8 @@ img {
   `;
 }
 
+// Set to ignore because testing it would imply testing fs, which isn't strictly necessary.
+/* istanbul ignore next */
```

Comment:
wait, why is this being ignored? It seems even easier to check than [`saveAssets`](https://github.com/GoogleChrome/lighthouse/blob/b1e091957e045a38b71d077eb6b161cfd394c02a/test/src/lib/asset-saver.js#L51-L63)